plugins/lsp/default.nix

```
@@ -90,6 +90,7 @@ in {
         "terraformls"
         "texlab"
         "tsserver"
+        "typos-lsp"
         "typst-lsp"
```

```suggestion
        "tsserver"
        "typst-lsp"
```

This is for deprecation warnings, as this is a new server, no need to mention it here.